backend/clubs/models.py

```diff
@@ -428,6 +428,34 @@ class Testimonial(models.Model):
     def __str__(self):
         return self.text
 
+class RecurringEvent(models.Model):
+    """
+    Represents recurring events hosted by a club.
+    """
+    id = models.IntegerField(primary_key=True)
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
+    offset = models.IntegerField()
+
+    """
+    RECURRING_DATES = (
+        ('m', 'Monday'),
+        ('t', 'Tuesday'),
+        ('w', 'Wednesday'),
+        ('th', 'Thursday'),
```

Comment:
not sure if we want to stick to Penn's standard that they use for courses, where the days are each one letter (`M,T,W,H,F,S,U`)